api/types.go

```diff
@@ -93,6 +93,36 @@ func TrackerStatusFromString(str string) TrackerStatus {
 	return TrackerStatusBug
 }
 
+func IsFilterValid(filter string) bool {
```

Comment:
Adding Queued and Error to TrackerStatus, this should be a oneliner like `TrackerStatusFromString(filter) != TrackerStatusBug`. Also, probably better to convert to TrackerStatus first and then call a ` func (st TrackerStatus) IsValid() bool`